js/server.js

```diff
@@ -38,7 +38,7 @@ var Server = function(config, callback) {
 			res.status(403).send("This device is not allowed to access your mirror. <br> Please check your config.js or config.js.sample to change this.");
 		});
 	});
-	app.use(helmet());
+	app.use(helmet(config.helmetOptions));
```

Comment:
If I'm thinking right here couldn't a user just add this to their module to block their module from being put in an iframe even if MM has frameguard disabled?

var frameguard = require('frameguard')
app.use(frameguard({ action: 'deny' }))

I'm no thinking they would but I suppose they could ... 

What would that do to MM if they do that an a user has frameguard disabled?